### PR TITLE
fix(onboarding): improve the cors settings to allow more onboarding

### PR DESCRIPTION
**Pull request by vamonte**
it's not a good idea to use localhost as allowed origins and it's not usefull in our case

### General

- [x] Write an explicit title for the Pull Request, following [Conventional Commits specification](https://www.conventionalcommits.org)
- [x] Test manually the implemented changes
- [x] Validate the code quality (indentation, syntax, style, simplicity, readability)

### Security

- [x] Consider the security impact of the changes made

### CODE REVIEW

File: django_forest/utils/cors.py [hunk 1]
```diff
@@ -16,15 +16,10 @@ def set_cors():
...
 
-    cors_allowed_origins = ['http://localhost:4200']
-    cors_allowed_origins.extend(get_list_setting('CORS_ALLOWED_ORIGINS'))
-
     cors_allowed_origins_regexes = [r'.*\.forestadmin\.com.*']
     cors_allowed_origins_regexes.extend(get_list_setting('CORS_ALLOWED_ORIGIN_REGEXES'))
 
-    settings.CORS_ALLOWED_ORIGINS = cors_allowed_origins
     settings.CORS_ALLOWED_ORIGIN_REGEXES = cors_allowed_origins_regexes
-    settings.CORS_URLS_REGEX = r'^/forest(/.*)?$'  # restrict it to /forest
```
Comment (GuillaumeCisco) [inline django_forest/utils/cors.py:27]: You will need this line.

Comment (vamonte) [reply django_forest/utils/cors.py:27]: I need it for what ?

Comment (vamonte) [reply django_forest/utils/cors.py:27]: It's just limit the urls for which the cors headers are sent. 
If  I have some routes that need these headers, it's won't work.
https://github.com/adamchainz/django-cors-headers#cors_urls_regex-str--patternstr

Comment (GuillaumeCisco) [reply django_forest/utils/cors.py:27]: Please look at the node.js and ruby SDK

**Review by matthv — APPROVED** [review]

**Pull request merged**